src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseAccessEndpoint.java

```diff
@@ -33,6 +35,7 @@ public CaseAccessEndpoint(CaseAccessOperation caseAccessOperation) {
         @ApiResponse(code = 200, message = "List of cases ids found"),
         @ApiResponse(code = 400, message = "Invalid case ID")
     })
+    @LogAudit(operationType = OperationType.SEARCH_CASE)
     public List<String> findCaseIdsGivenUserIdHasAccessTo(
```

Comment:
we said to ignore this endpoint. If we want to log it I don't think we should log it as a search. Search case should be only for the search api I'd say

Reply:
It is a mistake. V1 will be in another PR. Will remove from this PR.